cmd/admin-handlers.go

```diff
@@ -630,9 +630,40 @@ func isDryRun(qval url.Values) bool {
 	return false
 }
 
-type healObjectResult struct {
-	HealedCount  int
-	OfflineCount int
+// healResult - represents result of a heal operation like
+// heal-object, heal-upload.
+type healResult struct {
+	State healState `json:"state"`
+}
+
+// healState - different states of heal operation
+type healState int
+
+const (
+	// notHealed - none of the disks healed
+	notHealed healState = iota
```

Comment:
@abperiasamy :  By prefixing heal, there is namespace pollution. Use healOK, healFailed, healPartial. Always group them with common prefix to help IDEs autocomplete.